op-service/httputil/wrapped_response_writer.go

```diff
@@ -0,0 +1,38 @@
+package httputil
+
+import "net/http"
+
```

Comment:
I really liked this article on hn yesterday https://preslav.me/2022/08/09/i-dont-like-golang-default-http-handlers/

I like the idea of returning errors in http handlers like echo vs just returning early if there is an error

This seems like it would be a headache in the amount of changes to make, but they do give an example on how to implement this without using echo.

Thoughts?

Reply:
I like what that article suggests, though this particularly piece of code is unrelated to that. We'll still need a `ResponseWriter`, and the wrapper tracks response size and status code so we can log them/add them to metrics.

Reply:
If we want to convert all of our HTTP handlers to this pattern we should bring it up in OH and agree on it together.